.pax/prepare-workspace.sh

```diff
@@ -37,10 +31,6 @@ echo "[${SCRIPT_NAME}] cleaning PAX workspace ..."
 rm -fr "${PAX_WORKSPACE_DIR}/content"
 mkdir -p "${PAX_WORKSPACE_DIR}/content"
 
-# copy plugin definition files
-# echo "[${SCRIPT_NAME}] copying plugin definitions ..."
-# cp -r plugin-definition "${PAX_WORKSPACE_DIR}/content"
-
 # install peerDependencies
 echo "[${SCRIPT_NAME}] install peer dependencies (explorer-ui-server) ..."
 EXPLORER_UI_SERVER_VERSION=$(node -e "console.log(require('./package.json').peerDependencies['explorer-ui-server'])")
```

Comment:
We don't need this section with `npm install "explorer-ui-server@${EXPLORER_UI_SERVER_VERSION}" --no-save`. But we may need it for integration test.

Reply:
hmm, how to preserve for one?

Reply:
We can move this part to prepare-fvt.sh since this is not a necessary step to prepare pax. But for integration test, we need it to be installed to start ui server.

Reply:
downloading explorer-ui-server in prepare-fvt